stack.yaml

```diff
@@ -0,0 +1,15 @@
+resolver: lts-11.8
+
+packages:
+- '.'
+
+extra-deps:
+  - gtk-0.14.9
+  - gio-0.13.5.0
```

Comment:
Could this be fixed by updating gtk and gio in stackage?

Reply:
currently `gtk` and `gio` are not part of the LTS snapshot, so I've added them as extra deps here. This shouldn't cause a problem, should it?

Reply:
Only that the version numbers are likely to become stale here.  If you send a pull request to stackage we might be able to leave them out of this file.  I have the same problem with [leksah's stack.yaml file](https://github.com/leksah/leksah/blob/master/stack.yaml#L12).